wallet/src/de/schildbach/wallet/ui/backup/BackupWalletDialogFragment.java

```diff
@@ -111,6 +118,7 @@ public void onChanged(final Wallet wallet) {
                                     final String targetProvider = WalletUtils.uriToProvider(uri);
                                     final String password = passwordView.getText().toString().trim();
                                     checkState(!password.isEmpty());
+                                    checkState(backupWallet != null);
```

Comment:
In line 113, you'd observe that `backupWallet` which is then guaranteed to be non-null so you can remove this assert.